.travis.yml

```diff
@@ -26,9 +26,10 @@ script:
   - NOCONFIGURE=1 ./autogen.sh
   - >
     if [ -n "$MINGW" ]; then
+      unset CC CXX;
```

Comment:
This might fix mingw build

Reply:
Though, did you have a good reason to drop it?  I think it was required and didn't have anything to do with GTK2.